app/services/analytics_events.rb

```diff
@@ -6489,8 +6660,7 @@ def recaptcha_verify_result_received(
     score_threshold:,
     evaluated_as_valid:,
     form_class:,
-    exception_class:,
-    recaptcha_action:,
+    recaptcha_action:, exception_class: nil,
```

Comment:
I wouldn't expect this change.

```suggestion
    exception_class:,
    recaptcha_action:,
```

Reply:
You are right. Restored original with commit 03ace69ebc945ea11273e4eaa621fcd474dc0510